src/components/container/sidebar/JobList.tsx

```diff
@@ -7,7 +7,7 @@ import { TJobItem } from '@/lib/types';
 type JobListProps = {
   jobItems: TJobItem[];
   loading: boolean;
-  errorMessage: string | undefined;
+  errorMessage?: string | undefined;
```

Comment:
since this prop is optional we can remove the undefined data type from this prop